bastionzero/service/targets/web.go

```diff
@@ -38,13 +39,14 @@ type CreateWebTargetResponse struct {
 
 // ModifyWebTargetRequest is used to modify a Web target
 type ModifyWebTargetRequest struct {
-	TargetName    *string `json:"targetName,omitempty"`
-	ProxyTargetID *string `json:"proxyTargetId,omitempty"`
-	RemoteHost    *string `json:"remoteHost,omitempty"`
-	RemotePort    *Port   `json:"remotePort,omitempty"`
-	LocalPort     *Port   `json:"localPort,omitempty"`
-	LocalHost     *string `json:"localHost,omitempty"`
-	EnvironmentID *string `json:"environmentId,omitempty"`
+	TargetName         *string `json:"targetName,omitempty"`
+	ProxyTargetID      *string `json:"proxyTargetId,omitempty"`
+	ProxyEnvironmentID string  `json:"proxyEnvironmentId,omitempty"`
```

Comment:
Make this a pointer type. See other review comment in `ModifyDatabaseTargetRequest`

Reply:
[c909be9](https://github.com/bastionzero/bastionzero-sdk-go/pull/42/commits/c909be9fd7879761e2822e9efbbc2c3e6c8e90cb)